### PR TITLE
Add error to job - in progress

### PR DESCRIPTION
**Pull request by ArvindHarinder1**

### CODE REVIEW

**Comment by ArvindHarinder1** [comment]
#sign-off

**Comment by ArvindHarinder1** [comment]
@Lauragra  this is ready to merge.

File: api-reference/beta/resources/synchronization-synchronizationstatus.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ Represents the current status of the [synchronizationJob](synchronization-synchr
 |steadyStateLastAchievedTime|DateTimeOffset|The time when steady state (no more changes to the process) was last achieved. The Timestamp type represents date and time information using ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2014 would look like this: `'2014-01-01T00:00:00Z'`.|
 |synchronizedEntryCountByType|[stringKeyLongValuePair](synchronization-stringkeylongvaluepair.md) collection|Count of synchronized objects, listed by object type.|
 |troubleshootingUrl|String|In the event of an error, the URL with the troubleshooting steps for the issue.|
+|error|[quarantineError](synchronization-synchronizationerror.md)|Describes the error(s) putting the job into quarantine.|
```
Comment (Lauragra) [inline api-reference/beta/resources/synchronization-synchronizationstatus.md]: Should error be added as a possible value for the **code** property? The **quarantineError** type does not match the file that it is linking to. It looks like you're adding error as a value to the status code so not sure that it belongs in the Properties table.

File: concepts/changelog.md [hunk 2]
```diff
@@ -64,6 +64,7 @@ For details about known issues with Microsoft Graph APIs, see [Known issues](kno
 | **Change type** | **Version**   | **Description**                          |
 |:---|:---|:---|
 | Addition | beta and v1.0 | Added the **latestSupportedTlsVersion** property to the [subscription](/graph/api/resources/subscription?view=graph-rest-1.0) entity. Specifies the latest TLS version that the notification endpoint supports. Allows subscribers to use a deprecated version of TLS for a limited period. For details, see [Microsoft Graph subscriptions - deprecating TLS 1.0 and 1.1](https://developer.microsoft.com/graph/blogs/microsoft-graph-subscriptions-deprecating-tls-1-0-and-1-1/). |
+|Addition | beta | [synchronizationError](/api-reference/beta/resources/synchronization-synchronizationError.md) resource added to the provisioning job |
```
Comment (Lauragra) [inline concepts/changelog.md]: Please clarify what the update is here. This PR doesn't include the synchronizationError resource topic. Is this the addition of a new property or value on synchronizationStatus?

Comment (ArvindHarinder1) [reply concepts/changelog.md]: There is an existing synchronization error entity. This is adding that entity to synchronization quarantine where ass previously the entity was only applicable to synchronization task.

Comment (ArvindHarinder1) [reply concepts/changelog.md]: API review - https://microsoftgraph.visualstudio.com/onboarding/_git/onboarding/pullrequest/3060

Comment (ArvindHarinder1) [reply concepts/changelog.md]: Existing resource that is being added to the quarantine entity. https://docs.microsoft.com/en-us/graph/api/resources/synchronization-synchronizationerror?view=graph-rest-beta

**Pull request merged**